packages/shortest/README.md

```diff
@@ -73,6 +79,42 @@ shortest('Login to the app using username and password', {
 });
 ```
 
+## Test Dependencies
+You can create reusable test steps and chain them together. This helps organize complex test flows and maintain state between related tests:
+
+```typescript
+import { shortest, after } from '@antiwork/shortest';
+
+// Define reusable test steps
+const login = shortest('User can login', { 
+  email: 'test@example.com',
+  password: 'password'
+});
+
+const settings = shortest('Click settings button');
+
+// Chain tests with dependencies
+after(login, settings)
+  .shortest('Change language to Arabic')
+  .expect('Language should be Arabic');
```

Comment:
Do we need the expects? Feels like something shortest should handle.

Reply:
if there is no expectations we don't add them to the prompt, but we can specify it in the shortest string. it's just a preference.
![image](https://github.com/user-attachments/assets/23f8c03e-94f2-4ae1-913f-61564f2062a0)